telegram/ext/_dispatcher.py

```diff
@@ -102,13 +102,15 @@ class Dispatcher(Generic[BT, CCT, UD, CD, BD, JQ, PT]):
 
     Attributes:
         bot (:class:`telegram.Bot`): The bot object that should be passed to the handlers.
-        update_queue (:obj:`Queue`): The synchronized queue that will contain the updates.
+        update_queue (:class:`queue.Queue`): The synchronized queue that will contain the updates.
         job_queue (:class:`telegram.ext.JobQueue`): Optional. The :class:`telegram.ext.JobQueue`
             instance to pass onto handler callbacks.
         workers (:obj:`int`, optional): Number of maximum concurrent worker threads for the
             ``@run_async`` decorator and :meth:`run_async`.
-        user_data (:obj:`defaultdict`): A dictionary handlers can use to store data for the user.
-        chat_data (:obj:`defaultdict`): A dictionary handlers can use to store data for the chat.
+        user_data (:class:`collections.defaultdict`): A dictionary handlers can use to store dat
```

Comment:
```suggestion
        user_data (:class:`collections.defaultdict`): A dictionary handlers can use to store data
```
at this point only a spell check CI will save us 😆

Reply:
the horror 😬